docfx_project/ziti/overview.md

```diff
@@ -2,6 +2,15 @@
 uid: zitiOverview
 ---
 
+# Why?
+
+* You want to provide business value to your project as fast as possible but keep having to worry about security and networking.
+* ‘Secure networking’ is an oxymoron. There are no secure networks. Attacks are coming from the network, even from the local network your computer is on right now!
```

Comment:
replace the 'smart' apostrophes with straight quotes: ".

> Attacks are coming from the network
to?
"Attacks can come from any network"